### PR TITLE
Qumulo dontnet sdk

### PR DESCRIPTION
**Pull request by nikhgup**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

Details related to namespace approvals: https://github.com/Azure/azure-sdk/issues/5255 . Please check the approval in last comment

This is new PR for Qumulo SDK and we are abandoning the old PR here as it is having some issues and some files are not generated properly because of issue in swagger link copied https://github.com/Azure/azure-sdk-for-net/pull/33274

### CODE REVIEW

File: sdk/qumulo/Azure.ResourceManager.Qumulo/samples/autorest.tests.md
Comment (ArcturusZhang) [inline sdk/qumulo/Azure.ResourceManager.Qumulo/samples/autorest.tests.md:1]: Please run `dotnet build /t:GenerateTests` in this directory to generate the samples.

File: sdk/qumulo/Azure.ResourceManager.Qumulo/src/autorest.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+csharp: true
+library-name: Qumulo
+namespace: Azure.ResourceManager.Qumulo
+require: https://github.com/Azure/azure-rest-api-specs/blob/aceee26b40a3251a1dbcc76e57fb1e0860f1c76d/specification/liftrqumulo/resource-manager/readme.md
+output-folder: $(this-folder)/Generated
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
+  flatten-payloads: false
+
```
Comment (ArcturusZhang) [inline sdk/qumulo/Azure.ResourceManager.Qumulo/src/autorest.md:16]: Please add a `generate-model-factory: false` here to exclude the model factory.
The model factories are on our plan but not quite ready to release yet

File: sdk/qumulo/test-resources.json
Comment (ArcturusZhang) [inline sdk/qumulo/test-resources.json:1]: This file is unused by mgmt RPs, could we please remove this file?

File: sdk/qumulo/Azure.ResourceManager.Qumulo/tests/Azure.ResourceManager.Qumulo.Tests.csproj
Comment (ArcturusZhang) [inline sdk/qumulo/Azure.ResourceManager.Qumulo/tests/Azure.ResourceManager.Qumulo.Tests.csproj:1]: is it possible to include some test cases in your RP?

**Review by ArcturusZhang — COMMENTED** [review]
Please run `./eng/scripts/Export-API.ps1 qumulo` to generate the api file as well.

BTW has the namespace `Qumulo` been reviewed and approved by the arch board?

File: sdk/qumulo/Azure.ResourceManager.Qumulo/samples/autorest.tests.md
Comment (nikhgup) [reply sdk/qumulo/Azure.ResourceManager.Qumulo/samples/autorest.tests.md:1]: > Please run `./eng/scripts/Export-API.ps1 qumulo` to generate the api file as well.
> 
> BTW has the namespace `Qumulo` been reviewed and approved by the arch board?

yes please check here https://github.com/Azure/azure-sdk/issues/5255

File: sdk/qumulo/test-resources.json
Comment (nikhgup) [reply sdk/qumulo/test-resources.json:1]: removed

File: sdk/qumulo/Azure.ResourceManager.Qumulo/tests/Azure.ResourceManager.Qumulo.Tests.csproj
Comment (nikhgup) [reply sdk/qumulo/Azure.ResourceManager.Qumulo/tests/Azure.ResourceManager.Qumulo.Tests.csproj:1]: will create an item for it.

File: sdk/qumulo/Azure.ResourceManager.Qumulo/src/autorest.md (hunk 1 above)
Comment (nikhgup) [reply sdk/qumulo/Azure.ResourceManager.Qumulo/src/autorest.md:17]: done

File: sdk/qumulo/Azure.ResourceManager.Qumulo/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("Qumulo")]
```
Comment (ArthurMa1978) [inline sdk/qumulo/Azure.ResourceManager.Qumulo/src/Properties/AssemblyInfo.cs:6]: This  should be name of ARM resource, I guess it's `Qumulo.Storage` ?

Comment (nikhgup) [reply sdk/qumulo/Azure.ResourceManager.Qumulo/src/Properties/AssemblyInfo.cs:6]: @ArthurMa1978 yes arm resource name is like this "acctest8090/providers/Qumulo.Storage/fileSystems/nikhgup-test-1". do i need to just change the name in this file or update the namespace/folder structure of project as well

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**